src/cls/rbd/cls_rbd_client.h

```diff
@@ -119,6 +119,13 @@ namespace librbd {
                              std::vector<uint64_t> *sizes,
                              std::vector<parent_info> *parents,
                              std::vector<uint8_t> *protection_statuses);
+    void snap_timestamp_list_start(librados::ObjectReadOperation *op,
```

Comment:
I just follow the style of "snap_namespace_list"... But I agree, snapshot_XX is better, let me change it.